### ISSUE TITLE
Error with _hashlib with testing and running server.

### ISSUE THREAD

**Issue by ejcramer** [ripeta/repeat-aft#88]
With repeat-aft service (Python Django Rest Framework) on GCloud compute engine running Ubuntu 16.04.3 LTS,  I get this _hashlib error everytime I start it up (see attached text file).  Also happens when running
`./setup.py test`

The server still starts and works fine, but displays errors on the ssh console.

[hashlib-error.txt](https://github.com/ripeta/repeat-aft/files/1379973/hashlib-error.txt)
 
I ran this update successfully, but still get the same errors afterwards:
```
$ sudo apt-get update
$ sudo apt-get install libffi-dev g++ libssl-dev
$ sudo apt-get install build-essential libssl-dev libffi-dev python3-dev
```
If I try to install cryptography within the virtualenv, I get the same error.  It does not appear to install.
`pip3 install cryptography`

No other obvious answers online.  Will keep investigating this. 

P.S. This does not happen on MacOS install

**Comment by stevendborrelli** [comment]
Checking...

**Comment by ejcramer** [comment]
Steven could not reproduce this error, so I concluded it was unique to my setup on gcloud.
I did find some system python libraries thought they were 3.6.3 and my virtualenv was 3.6.2.  So I wiped out the virtualenv.  Ran apt-get update && apt-get upgrade, reinstalled all python pre-requisites for repeat-aft and recreated a virtualenv.  I was able to get the service started without the _hashlib error and the service is working well using the pre-existing database.  Guess I should have done that sooner!
